index.js

```diff
@@ -0,0 +1,86 @@
+const express = require('express');
+const knex = require('knex');
+const knexConfig = require('./knexfile').development;
+
+const db = knex(knexConfig);
+
+const server = express();
+
+server.use(express.json());
+
+server.get('/api/cohorts', async (req, res) => {
+  try {
+    const cohorts = await db('cohort');
+    res.status(200).json(cohorts);
+  } catch (error) {
+    res.status(500).json(error);
+  }
+});
+
+server.get('/api/cohorts/:id', async (req, res) => {
```

Comment:
Hey borja, how can you take out the `api/cohorts` from all the paths? use express router